src/Drupal/Commands/core/MigrateRunnerCommands.php

```diff
@@ -543,6 +562,25 @@ public function messages(string $migrationId): RowsOfFields
             $row['source_ids'] = implode(' : ', $sourceIds);
             $row['destination_ids'] = implode(' : ', $destinationIds);
 
+            // Only show relevant messages.
+            // TODO This logic can be simplified if a new \FilterIterator child
```

Comment:
We have something similar I guess. Isn't `MigrateIdMapFilter` something that we can use here?

Reply:
Right, different name but similar. I will change the code to use it.